Header header

bool is_armed
bool is_connected
bool gps_ok
bool imu_ok
float32 battery_percent
float32 battery_voltage
float32 temperature
string flight_mode
